wvdecrypter/cdm/media/cdm/cdm_adapter.cc

```diff
@@ -63,10 +64,18 @@ void* GetCdmHost(int host_interface_version, void* user_data)
 
 }  // namespace
 
+std::atomic<bool> exit_thread_flag{false};
```

Comment:
For simpler types regular assignment is easier to understand.

`std::atomic<bool> exit_thread_flag = false;`

Reply:
Moved initialization to "Initialize()" as suggestion does not compile.

Reply:
Ah, of course. It's atomic bool. My bad.